### PR TITLE
Reuse the configuration of the gradle-nexus-staging-plugin

### LINKED ISSUE #3: Feature Request: Reuse the stagingProfileId of `io.codearte.nexus-staging`
This plugin should detect if the `io.codearte.nexus-staging` plugin is also applied (on the root project) and reuse its `nexusStaging.*` properties:

- `nexusStaging.stagingProfileId` -> `nexusPublish.stagingProfileId`
- `nexusStaging.username` -> `nexusPublish.username`
- `nexusStaging.password` -> `nexusPublish.password`
- `nexusStaging.packageGroup` -> `nexusPublish.packageGroup`

### PR DESCRIPTION
**Pull request by larsgrefer**
fixes #3

### CODE REVIEW

**Review by marcphilipp — CHANGES_REQUESTED** [review]
Awesome, thanks!

Could you add a integration test that applies both plugins and checks the wiring works correctly? So far there's only a single test but you should be able to use it as reference.

Moreover, we should document this (in the readme for now).

**Comment by larsgrefer** [comment]
Sure, I'll add some tests

**Comment by larsgrefer** [comment]
@marcphilipp Please review again

File: README.md [hunk 1]
```diff
@@ -35,3 +35,14 @@ nexusPublishing {
     password = "your-password" // defaults to project.properties["nexusPassword"]
 }
 ```
+
+If the [`io.codearte.nexus-staging` plugin]() is applied on the root project, the following default values change:
+```groovy
```
Comment (marcphilipp) [inline README.md]:
```suggestion
```gradle
```

**Review by marcphilipp — APPROVED** [review]
Looks great except for one minor thing. 👍

File: README.md (hunk 1 above)
Comment (larsgrefer) [reply README.md]: done 👍

File: src/test/java/de/marcphilipp/gradle/nexus/StagingPluginIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.marcphilipp.gradle.nexus;
+
+import io.codearte.gradle.nexus.NexusStagingExtension;
+import io.codearte.gradle.nexus.NexusStagingPlugin;
+import org.gradle.api.Project;
+import org.gradle.testfixtures.ProjectBuilder;
+import org.junit.Test;
```
Comment (marcphilipp) [inline src/test/java/de/marcphilipp/gradle/nexus/StagingPluginIntegrationTest.java]:
```suggestion
import org.junit.jupiter.api.Test;
```

**Review by marcphilipp — CHANGES_REQUESTED** [review]
As you can see on the [build scan](https://scans.gradle.com/s/ydreczfuxmhrm/tests), the test was not executed. Root cause is that it uses the wrong `@Test` annotation. I'll exclude `junit:junit` so this won't happen again.

File: src/test/java/de/marcphilipp/gradle/nexus/StagingPluginIntegrationTest.java (hunk 2 above)
Comment (larsgrefer) [reply src/test/java/de/marcphilipp/gradle/nexus/StagingPluginIntegrationTest.java]: done 👍

**Comment by larsgrefer** [comment]
The tests execute successfully now.

**Review by marcphilipp — APPROVED** [review]

**Comment by marcphilipp** [comment]
Thanks! 👍

**Pull request merged**